test/topology/conftest.py

```diff
@@ -5,7 +5,6 @@
 #
 # This file configures pytest for all tests in this directory, and also
 # defines common test fixtures for all of them to use
-import os
```

Comment:
failed on setup with "NameError: name 'os' is not defined"

Reply:
Added it in test/object_store/conftest.py where it actually should be, rather than in topology/conftest.py